hermes-management/src/main/java/pl/allegro/tech/hermes/management/domain/health/NodeDataProvider.java

```diff
@@ -0,0 +1,31 @@
+package pl.allegro.tech.hermes.management.domain.health;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+
+import javax.inject.Inject;
+import java.net.InetAddress;
+import java.net.UnknownHostException;
+
+@Component
+class NodeDataProvider {
+
+    private final String serverPort;
+
+    @Inject
```

Comment:
I think this annotation is not needed, but please check. :)

Reply:
Done